.github/workflows/cherry-pick.yml

```diff
@@ -1,9 +1,7 @@
 name: Cherry Pick Tool
 on:
     issues:
-        types: [milestoned, labeled]
-    pull_request:
-        types: [closed]
+        types: [milestoned]
```

Comment:
Shouldn't the `types: [milestoned]` be for `pull_request` instead of `issues`?

Reply:
So I was also under the same assumption, but the milestone is only supported under issues, and github treats PRs are issue under the hood for some cases.